src/app/pages/applications/chart-releases/chart-releases.component.ts

```diff
@@ -82,6 +92,16 @@ export class ChartReleasesComponent implements OnInit {
     this.minioForm = new MinioFormComponent(this.mdDialog,this.dialogService,this.modalService);
   }
 
+  addChartReleaseChangedEventListner() {
+    this.chartReleaseChangedListener = this.ws.subscribe("chart.release.query").subscribe((evt) => {
+      this.chartItems.forEach(app => {
```

Comment:
Do you think if we should use an object instead of looping each time on a list for each event ?